frontends/election-manager/src/app.test.tsx

```diff
@@ -139,29 +138,30 @@ async function authenticateWithAdminCard(card: MemoryCard) {
   card.insertCard(
     makeAdminCard(eitherNeitherElectionDefinition.electionHash, '123456')
   );
-  jest.advanceTimersByTime(CARD_POLLING_INTERVAL);
   await screen.findByText('Enter the card security code to unlock.');
-  fireEvent.click(screen.getByText('1'));
-  fireEvent.click(screen.getByText('2'));
-  fireEvent.click(screen.getByText('3'));
-  fireEvent.click(screen.getByText('4'));
-  fireEvent.click(screen.getByText('5'));
-  fireEvent.click(screen.getByText('6'));
+  userEvent.click(screen.getByText('1'));
+  userEvent.click(screen.getByText('2'));
+  userEvent.click(screen.getByText('3'));
+  userEvent.click(screen.getByText('4'));
+  userEvent.click(screen.getByText('5'));
+  userEvent.click(screen.getByText('6'));
   await screen.findByText('Remove card to continue.');
   card.removeCard();
-  jest.advanceTimersByTime(CARD_POLLING_INTERVAL);
```

Comment:
I've started removing some of these `jest.advanceTimersByTime(CARD_POLLING_INTERVAL)` calls, specifically in UI layer tests where we can just rely on `await screen.findBy` calls

Reply:
Will that make the tests take significantly longer to execute?

Reply:
Good q! The `CARD_POLLING_INTERVAL` is so short (100ms), so it seemed unlikely to me (I know we could one day increase the `CARD_POLLING_INTERVAL`, but that seems unlikely since it'd be to the detriment of user experience). Nonetheless, I conducted some science just to be sure 🧑‍🔬 

I ran app.test.tsx with the `jest.advanceTimersByTime(CARD_POLLING_INTERVAL)` calls and without and saw no difference:

```
With jest.advanceTimersByTime | Without jest.advanceTimersByTime
----------------------------------------------------------------
7.819s                        | 7.237s
6.302s                        | 6.498s
6.147s                        | 6.329s
6.922s                        | 6.976s
6.421s                        | 6.240s
6.513s                        | 5.969s
----------------------------------------------------------------
6.687s average                | 6.542s average
```

Reply:
Awesome, you should tell the whole team! This will make tests much cleaner

Reply:
Good call - will do!